testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -751,6 +720,21 @@ class _GromacsReader_offsets(TestCase):
     ref_volume = 362270.0
     ref_offsets = None
     _reader = None
+    prec = 3
+    
+    @pytest.fixture()
```

Comment:
I think, this must be a function/method level-fixture. It *cannot* be re-used for the whole class. I don't know if this is its default scope. — @kain88-de please correct me if I am wrong here.

The reason is that MDAnalysis creates a hidden file when it reads the trajectory the first time (the so-called offsets file). This hidden file is stored in the same directory as the trajectory file. The tests here are supposed to test what happens when this hidden file does not exist yet.

Reply:
@orbeckst by default things are function scoped